[ECS] Weird errors couting

TLDR: i see many errors in output, but ECS says `[ERROR] Found 2 errors.` which seems to be incorrect.

Is it like 2 errors repeating in multiple files? 2 errors in total? More clarification would be awesome.

I would love to propose a text change but at this moment i do not understand what 2 errors exactly means :-) 

Here is full output:
```
> vendor/bin/ecs check --config easy-coding-standard.neon src app --ansi
 18/18 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


1) app/ApplicationModule/Presenters/BasePresenter.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -6,7 +6,7 @@
 use WebLoader\Nette\CssLoader;
 use WebLoader\Nette\JavaScriptLoader;
 
-abstract class BasePresenter extends AppBasePresenter
+abstract class AbstractBasePresenter extends AppBasePresenter
 {
        protected function createComponentCss(): CssLoader
        {

    ----------- end diff -----------

Applied checkers:

 - Symplify\CodingStandard\Sniffs\Naming\AbstractClassNameSniff



2) app/Components/BaseControl.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -9,7 +9,7 @@
 use Nette\Application\UI\Presenter;
 use Nette\Reflection\ClassType;
 
-abstract class BaseControl extends Control
+abstract class AbstractBaseControl extends Control
 {
        use AutowireComponentFactories;
 
@@ -58,8 +58,8 @@
                $controlReflection = new ClassType(get_class($this));
                $controlDir = dirname($controlReflection->getFileName());
                $template->setFile(
-                       $controlDir . DIRECTORY_SEPARATOR . 'templates' . DIRECTORY_SEPARATOR . ($filename ?: "${view}.latte" ) 
+                       $controlDir . DIRECTORY_SEPARATOR . 'templates' . DIRECTORY_SEPARATOR . ($filename ?: "${view}.latte" )
                );
                return $template;
        }
 }

    ----------- end diff -----------

Applied checkers:

 - PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\SuperfluousWhitespaceSniff.EndLine
 - Symplify\CodingStandard\Sniffs\Naming\AbstractClassNameSniff


    ---------- begin diff ----------
--- Original
+++ New
@@ -14,12 +14,12 @@
        use AutowireComponentFactories;
 
        /**
-        * @var array of function (BaseControl, Nette\ComponentModel\Container $parent)
+        * @var array of function (BaseControl, Nette\ComponentModel\Container)
         */
        public $onAttached = [];
 
        /**
-        * @var array of function (BaseControl, Nette\App\UI\Presenter $presenter)
+        * @var array of function (BaseControl, Nette\App\UI\Presenter)
         */
        public $onPresenterAttached = [];
 

    ----------- end diff -----------

Applied checkers:

 - PhpCsFixer\Fixer\Phpdoc\PhpdocVarWithoutNameFixer



3) app/Components/GoogleAnalytics/IControlFactory.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -2,8 +2,8 @@
 
 namespace App\Components\GoogleAnalytics;
 
-interface IControlFactory
+interface ControlFactoryInterface
 {
        /** @return Control */
        function create();
 }

    ----------- end diff -----------

Applied checkers:

 - Symplify\CodingStandard\Sniffs\Naming\InterfaceNameSniff



4) app/Presenters/BasePresenter.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -8,7 +8,7 @@
 use Nette\Application\UI\Presenter;
 use WebLoader\Nette\LoaderFactory;
 
-abstract class BasePresenter extends Presenter
+abstract class AbstractBasePresenter extends Presenter
 {
        use AutowireProperties;
        use AutowireComponentFactories;

    ----------- end diff -----------

Applied checkers:

 - Symplify\CodingStandard\Sniffs\Naming\AbstractClassNameSniff


    ---------- begin diff ----------
--- Original
+++ New
@@ -23,8 +23,7 @@
         * @var LoaderFactory
         * @autowire
         */
-       protected $webLoader;
-
+       protected $loaderFactory;
 
        protected function startup(): void
        {

    ----------- end diff -----------

Applied checkers:

 - PhpCsFixer\Fixer\ClassNotation\ClassAttributesSeparationFixer
 - Symplify\CodingStandard\Fixer\Naming\PropertyNameMatchingTypeFixer



5) app/Presenters/ErrorPresenter.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -2,8 +2,9 @@
 
 namespace App\Presenters;
 
-use App\ApplicationModule\Presenters\BasePresenter as ApplicationBasePresenter,
-       Nette;
+use App\ApplicationModule\Presenters\BasePresenter as ApplicationBasePresenter;
+use Nette,
+       Nette\Application\BadRequestException;
 use Tracy\Debugger;
 
 
@@ -15,11 +16,14 @@
                        $this->payload->error = TRUE;
                        $this->terminate();
 
-               } elseif ($exception instanceof Nette\Application\BadRequestException) {
+               } elseif ($exception instanceof BadRequestException) {
                        $code = $exception->getCode();
                        // load template 403.latte or 404.latte or ... 4xx.latte
                        $this->setView(in_array($code, [403, 404, 405, 410, 500], true) ? $code : '4xx');
                        // log to access.log
-                       Debugger::log("HTTP code ${code}: {$exception->getMessage()} in {$exception->getFile()}:{$exception->getLine()}", 'access');
+                       Debugger::log(
+                               "HTTP code ${code}: {$exception->getMessage()} in {$exception->getFile()}:{$exception->getLine()}",
+                               'access'
+                       );
 
                } else {

    ----------- end diff -----------

Applied checkers:

 - PhpCsFixer\Fixer\Import\OrderedImportsFixer
 - Symplify\CodingStandard\Fixer\Import\ImportNamespacedNameFixer
 - Symplify\CodingStandard\Fixer\LineLength\BreakMethodCallsFixer



6) src/Templating/OrphansFilter.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -6,8 +6,8 @@
 
 class OrphansFilter
 {
-       public function __invoke(FilterInfo $info, string $content): string
+       public function __invoke(FilterInfo $filterInfo, string $content): string
        {
                return preg_replace('~( +(a|cca|č.|či|do|i|k|ke|na|o|od|po|s|tj.|u|v|z|za) +)~iu', ' $2&nbsp;', $content);
        }
 }

    ----------- end diff -----------

Applied checkers:

 - Symplify\CodingStandard\Fixer\Naming\PropertyNameMatchingTypeFixer



 ------ -------------------------------------------------------------------------------- 
  Line   app/Components/BaseControl.php                                                  
 ------ -------------------------------------------------------------------------------- 
  58     Use service and constructor injection rather than instantiation with            
         "new Nette\Reflection\ClassType".                                               
         (Symplify\CodingStandard\Sniffs\DependencyInjection\NoClassInstantiationSniff)  
 ------ -------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   app/Presenters/ErrorPresenter.php                                     
 ------ ---------------------------------------------------------------------- 
  15     Property "$payload" should be used instead of dynamically defined     
         one.                                                                  
         (Symplify\CodingStandard\Sniffs\Property\DynamicPropertySniff)        
 ------ ---------------------------------------------------------------------- 

                                                                                
 [ERROR] Found 2 errors.                                                        
                                                                                

                                                                                
 [OK] 8 files are fixable! Just add "--fix" to console command and rerun to     
      apply.                                                                    
                                                                                

Script vendor/bin/ecs check --config easy-coding-standard.neon src app --ansi handling the cs event returned with error code 1
```

--- a/packages/EasyCodingStandard/bin/ecs
+++ b/packages/EasyCodingStandard/bin/ecs
@@ -1,8 +1,8 @@
 #!/usr/bin/env php
 <?php declare(strict_types=1);
 
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
 use Symfony\Component\DependencyInjection\Container;
 use Symplify\EasyCodingStandard\Console\Application;
 use Symplify\PackageBuilder\Console\Style\SymfonyStyleFactory;
@@ -15,8 +15,8 @@ try {
     $container = require __DIR__ . '/ecs-container.php';
 
     $application = $container->get(Application::class);
-    $input = $container->get(InputInterface::class);
-    $output = $container->get(OutputInterface::class);
+    $input = $container->get(ArgvInput::class);
+    $output = $container->get(ConsoleOutput::class);
     exit($application->run($input, $output));
 } catch (Throwable $throwable) {
     (SymfonyStyleFactory::create())->error($throwable);

--- a/packages/EasyCodingStandard/src/Console/Command/CheckCommand.php
+++ b/packages/EasyCodingStandard/src/Console/Command/CheckCommand.php
@@ -165,8 +165,9 @@ final class CheckCommand extends Command
     {
         if ($errorCount) {
             $this->easyCodingStandardStyle->error(sprintf(
-                'Found %d error%s.',
+                'Found %d error%s that need%s to be fixed manually.',
                 $errorCount,
+                $errorCount === 1 ? '' : 's',
                 $errorCount === 1 ? '' : 's'
             ));
         }
@@ -176,7 +177,8 @@ final class CheckCommand extends Command
         }
 
         $this->easyCodingStandardStyle->success(sprintf(
-            '%d file%s %s fixable! Just add "--fix" to console command and rerun to apply.',
+            '%s%d file%s %s fixable! Just add "--fix" to console command and rerun to apply.',
+            $errorCount ? 'Good news is that ': '',
             $fileDiffsCount,
             $fileDiffsCount === 1 ? '' : 's',
             $fileDiffsCount === 1 ? 'is' : 'are'

--- a/packages/EasyCodingStandard/src/Console/Command/CheckCommand.php
+++ b/packages/EasyCodingStandard/src/Console/Command/CheckCommand.php
@@ -178,7 +178,7 @@ final class CheckCommand extends Command
 
         $this->easyCodingStandardStyle->success(sprintf(
             '%s%d file%s %s fixable! Just add "--fix" to console command and rerun to apply.',
-            $errorCount ? 'Good news is that ': '',
+            $errorCount ? 'Good news is that ' : '',
             $fileDiffsCount,
             $fileDiffsCount === 1 ? '' : 's',
             $fileDiffsCount === 1 ? 'is' : 'are'

--- a/packages/EasyCodingStandard/src/config/external-services.yml
+++ b/packages/EasyCodingStandard/src/config/external-services.yml
@@ -6,12 +6,14 @@ services:
     Symfony\Component\Console\Terminal: ~
 
     # Console
-    Symfony\Component\Console\Input\ArgvInput: ~
+    Symfony\Component\Console\Input\ArgvInput:
+        public: true
 
-    Symfony\Component\Console\Output\ConsoleOutput: ~
+    Symfony\Component\Console\Output\ConsoleOutput:
+        public: true
+
     Symfony\Component\Console\Output\OutputInterface:
         alias: Symfony\Component\Console\Output\ConsoleOutput
-        public: true
 
     Symfony\Component\Console\Style\SymfonyStyle: ~
 

--- a/packages/Statie/bin/statie
+++ b/packages/Statie/bin/statie
@@ -1,8 +1,8 @@
 #!/usr/bin/env php
 <?php declare(strict_types=1);
 
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
 use Symfony\Component\DependencyInjection\Container;
 use Symplify\PackageBuilder\Console\Style\SymfonyStyleFactory;
 use Symplify\Statie\Console\Application;
@@ -15,8 +15,8 @@ try {
     $container = require __DIR__ . '/statie-autoload.php';
 
     $application = $container->get(Application::class);
-    $input = $container->get(InputInterface::class);
-    $output = $container->get(OutputInterface::class);
+    $input = $container->get(ArgvInput::class);
+    $output = $container->get(ConsoleOutput::class);
     exit($application->run($input, $output));
 } catch (Throwable $throwable) {
     (SymfonyStyleFactory::create())->error($throwable);

--- a/packages/Statie/bin/statie
+++ b/packages/Statie/bin/statie
@@ -12,7 +12,7 @@ gc_disable();
 
 try {
     /** @var Container $container */
-    $container = require __DIR__ . '/statie-autoload.php';
+    $container = require __DIR__ . '/statie-container.php';
 
     $application = $container->get(Application::class);
     $input = $container->get(ArgvInput::class);

--- a/packages/Statie/src/config/external-services.yml
+++ b/packages/Statie/src/config/external-services.yml
@@ -7,12 +7,14 @@ services:
 
     Symplify\PackageBuilder\Parameter\ParameterProvider: ~
 
-    Symfony\Component\Console\Input\ArgvInput: ~
-    Symfony\Component\Console\Output\ConsoleOutput: ~
+    Symfony\Component\Console\Input\ArgvInput:
+        public: true
+
+    Symfony\Component\Console\Output\ConsoleOutput:
+        public: true
 
     Symfony\Component\Console\Output\OutputInterface:
         alias: Symfony\Component\Console\Output\ConsoleOutput
-        public: true
 
     Symfony\Component\EventDispatcher\EventDispatcher: ~
 

--- a/packages/TokenRunner/src/Wrapper/FixerWrapper/ClassWrapper.php
+++ b/packages/TokenRunner/src/Wrapper/FixerWrapper/ClassWrapper.php
@@ -119,7 +119,7 @@ final class ClassWrapper
             return null;
         }
 
-        $parentClassNameToken = array_pop($stringTokens);
+        $parentClassNameToken = array_shift($stringTokens);
 
         return $parentClassNameToken->getContent();
     }

